### PR TITLE
old boot.php functions replaced in various places

### PR DESCRIPTION
**Pull request by annando**
This is a follow up of PR https://github.com/friendica/friendica/pull/12020

### CODE REVIEW

File: doc/themes.md [hunk 1]
```diff
@@ -123,21 +123,21 @@ The selected 1st part will be saved in the database by the theme_post function.
 
     function theme_post(App $a){
         // non local users shall not pass
-        if (! local_user()) {
+        if (! Session::getLocalUser()) {
```
Comment (MrPetovan) [inline doc/themes.md]:
```suggestion
        if (!Session::getLocalUser()) {
```

File: doc/themes.md [hunk 2]
```diff
@@ -123,21 +123,21 @@ The selected 1st part will be saved in the database by the theme_post function.
...
+            DI::pConfig()->set(Session::getLocalUser(), 'duepuntozero', 'colorset', $_POST['duepuntozero_colorset']);
         }
     }
 
 Now that this information is set in the database, what should friendica do with it?
 For this, have a look at the theme.php file of the *duepunto zero*.
 There you'll find somethink alike
 
-        $colorset = DI::pConfig()->get( local_user(), 'duepuntozero','colorset');
+        $colorset = DI::pConfig()->get( Session::getLocalUser(), 'duepuntozero','colorset');
```
Comment (MrPetovan) [inline doc/themes.md]:
```suggestion
        $colorset = DI::pConfig()->get(Session::getLocalUser(), 'duepuntozero','colorset');
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: doc/themes.md (hunk 1 above)
Comment (MrPetovan) [inline doc/themes.md]:
```suggestion
        if (!Session::getLocalUser()) {
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by annando** [comment]
Thanks for having a look at the code beauty. This had been some "hardcore search & replace".

**Pull request merged**